Dockerfile

```diff
@@ -1,15 +1,11 @@
 FROM node:latest
```

Comment:
Should change the tag to a specific version of node, eg. v16, instead of 'latest'

Reply:
Will change to node:17